src/transformers/convert_graph_to_onnx.py

```diff
@@ -330,6 +334,29 @@ def convert(
         convert_tensorflow(nlp, opset, output)
 
 
+def optimize(onnx_model_path: Path) -> Path:
+    """
+    Load the model at the specified path and let onnxruntime look at transformations on the graph
+    to enable all the optimizations possible
+    Args:
+        onnx_model_path: filepath where the model binary description is stored
+
+    Returns: Path where the optimized model binary description has been saved
+
+    """
+    from onnxruntime import SessionOptions, InferenceSession
+
+    # Generate model name with suffix "optimized"
+    opt_model_path = generate_identified_filename(onnx_model_path, "-optimized")
+    sess_option = SessionOptions()
+    sess_option.optimized_model_filepath = opt_model_path.as_posix()
+    _ = InferenceSession(onnx_model_path.as_posix(), sess_option)
```

Comment:
LGTM. Just one thing, for transformer-based models, this method doesn't optimize TF as much as PyTorch.

Reply:
I can add a warning if `args.framework == "tf"` to let the. user know, wdyt ?

Reply:
That's great! Could we add more into to ask the user to try optimizing the TF model with onnxruntime_tools?

Reply:
I have improved the TF compatibility message and added a link to the onnxruntime_tools for Transformer based model 👍